ansible/roles/jenkins-worker/templates/aix.rc2.j2

```diff
@@ -17,6 +17,7 @@ start )
               export NODE_TEST_DIR=$HOME/tmp; \
               export NODE_COMMON_PIPE="$HOME/test.pipe"; \
               export OSTYPE=AIX72; \
```

Comment:
why do we set this to `AIX72`?

Reply:
I have no idea what `OSTYPE` is used for. Originally AIX 7.2 was what we were going to use on the current machines but we rolled back to AIX 7.1 for minimum supported version/compatibility reasons -- I suspect this wasn't adjusted when we switched to AIX 7.1.